### PR TITLE
Add challenge-count-the-vowels

### LINKED ISSUE #383: challenge-count-the-vowels
> **Please read [For Newbie](https://github.com/labex-labs/scenarios/issues/18) first. 请先阅读 [新手须知](https://github.com/labex-labs/scenarios/issues/18)。**

## Basic

> Basic information about the scenario, which can be changed if required.
> 实验/挑战的基本信息，如有需要，可以修改。

- **Suggest Title 建议标题**: Count the Vowels
- **Suggest Steps 建议步骤数**: 1
- **Suggest Image 建议环境**: `webide-ubuntu:2204`

## Sources

> Issues has provided the material; the part not provided must be completed by the author.
> Issues 已经提供了材料；没有提供的部分由作者自行补充完成。

- [x] Full Text 正文内容
- [x] Reference Answers 参考答案
- [x] Verify Script 检测脚本

## Skills

> Suggested skills to include can be modified as needed. **The main skill points that should be included in this section are bolded**. From [Skills Tree](https://labex-dev.feishu.cn/base/bascnNz4Nqjqgqm1Nm5AYke6xxb?table=tblV5pGIsGZMxmE9&view=vewZpkLWRp).
> 建议包括的 Skills ID，可以根据需要增删。**本章节应包含的主要技能点已加粗**，技能点来自 [技能树](https://labex-dev.feishu.cn/base/bascnNz4Nqjqgqm1Nm5AYke6xxb?table=tblV5pGIsGZMxmE9&view=vewZpkLWRp)

- **`python/function_basic`**
- **`python/if_else`**
- **`python/for_loop`**
- **`python/str`**

### PR DESCRIPTION
**Pull request by AsukaLove2**
# PR Checklist

- fix #383 

请替换上方 `ID` 为本 PR 相关的 Issue ID，如 `fix #18`。Issue ID 可以在 Issue 的 URL 中找到，如 `https://github.com/labex-labs/scenarios/issues/18`，则 ID 为 `18`。

请保留上方链接前的 `fix ` 关键字，确保 issue 可以在 PR 合并后自动关闭。

你的内容会被自动同步上线并生成测试课程，完成 [线上测试](https://www.labex.wiki/zh/basic/how-to-test) 后，请评论 "test completed"，以便我们知道你已完成测试。

感谢你提出 PR，为 LabEx 做出贡献！

---

Please replace `ID` with the relevant Issue ID for this PR, such as `fix #18`. The Issue ID can be found in the URL of the Issue, for example, `https://github.com/labex-labs/scenarios/issues/18`, where the ID is `18`.

Please keep the `fix ` keyword before the link to ensure the issue is automatically closed after the PR is merged.

Your content will be automatically synchronized online and a test course will be generated. After completing the [online test](https://www.labex.wiki/en/basic/how-to-test), please comment "test completed" so that we know you have finished the test.

Thank you for submitting a PR and contributing to LabEx!

### CODE REVIEW

File: python/challenge-count-the-vowels/finish.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Summary
+
+In this challenge, you were tasked with creating a Python function that counts the number of vowels in a given string. You were able to use your knowledge of string manipulation and Python functions to complete this challenge.
```
Comment (jiaosenvip) [inline python/challenge-count-the-vowels/finish.md:3]: 这一段内容修改一下，替换成下面内容。

```md
Through the challenge of creating a function to count vowels in a string, we learned about functions, parameters, strings, loops, conditionals, and the importance of counting and iterating through strings in programming. This exercise provided a practical application of these concepts.
```

File: python/challenge-count-the-vowels/index.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
+  "type": "challenge",
+  "title": "Count the vowels",
```
Comment (jiaosenvip) [inline python/challenge-count-the-vowels/index.json:3]: `Count the Vowels` 注意标题书写

File: python/challenge-count-the-vowels/step1.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Count the Vowels
+
+Create a function called `count_vowels` in `count_vowels.py` that takes in one parameter, a string. The function should count the number of vowels in the string and return it as an integer.
+
```
Comment (jiaosenvip) [inline python/challenge-count-the-vowels/step1.md:4]: Requirements 前面增加 TODO

```
## TODO

To complete `count_vowels.py`.
```

**Review by jiaosenvip — CHANGES_REQUESTED** [review]
注意标题大小写，增加TODO，修改总结。

**Review by jiaosenvip — APPROVED** [review]

**Review by huhuhang — APPROVED** [review]

**Pull request merged**